resources/assets/coffee/react/profile-page/medals.coffee

```diff
@@ -32,6 +33,15 @@ ProfilePage.Medals = React.createClass
 
     _.includes @_achieved, id
 
+  _getDateAchieved: (id) ->
```

Comment:
>at this point it's easier to just pass the whole user's own achievement for the particular medal.

does it really make sense though? we're duplicating passed data here as the only thing we need is the actual date, and the achievement data is already being passed.

also, i don't see the users achievements stored anywhere easily accessible, so i have to retrieve them in pretty much the same fashion as i do now with dates.